lib/realtime_signs.ex

```diff
@@ -21,11 +21,12 @@ defmodule RealtimeSigns do
         Engine.Departures,
         Engine.Static,
         Engine.Alerts,
+        Engine.BusPredictions,
```

Comment:
Since these processes will be deployed in dev (and prod?) first without actually being used, should we add any logging so that we have a way to verify them as running? Or do you think it's reasonable to wait until Thursday to see

Reply:
Hmm yeah that's fair. I don't think there's any reason to doubt that the processes will start up improperly so no need to add any logging there. And agreed, we probably don't want to add noise to the logs. I think we're good to just watch for a lack of errors and see how things go Thursday then!